src/Microsoft.OData.Core/UriParser/Aggregation/AggregationMethod.cs

```diff
@@ -9,7 +9,31 @@ namespace Microsoft.OData.UriParser.Aggregation
     /// <summary>
     /// Enumeration of methods used in the aggregation clause
     /// </summary>
-    public enum AggregationMethod
+    public sealed class AggregationMethod
+    {
+        public AggregationMethodType MethodType { get; private set; }
+        public string MethodLabel { get; private set; }
+
+        private AggregationMethod(AggregationMethodType aggregationMethodType)
+        {
+            this.MethodType = aggregationMethodType;
+        }
+
+        public static AggregationMethod Sum = new AggregationMethod(AggregationMethodType.Sum);

```

Comment:
Seems like the standard methods should have names too.  Ah, but what name, English or localized?.  On second thought, leave the label blank for standard methods.